sql/core/src/main/scala/org/apache/spark/sql/execution/bucketing/CoalesceBucketsInJoin.scala

```diff
@@ -131,27 +129,11 @@ object ExtractJoinWithBuckets {
     }
   }
 
-  /**
-   * The join keys should match with expressions for output partitioning. Note that
-   * the ordering does not matter because it will be handled in `EnsureRequirements`.
-   */
-  private def satisfiesOutputPartitioning(
```

Comment:
Just a question. Is this moved to Join without any change?

Reply:
One more thing is delete comment: "Note that the ordering does not matter because it will be handled in `EnsureRequirements`."